flatcar-linux/kubernetes/workers.tf

```diff
@@ -10,8 +10,10 @@ module "workers" {
   os_version             = var.os_version
 
   # machine
-  name   = var.workers[count.index].name
-  mac    = var.workers[count.index].mac
+  name            = var.workers[count.index].name
+  mac             = var.workers[count.index].mac
+  extra_selectors = (length(var.workers[count.index].extra_selectors) > 0) ? join("", [for key, value in var.workers[count.index].extra_selectors : "${urlencode(key)}=${urlencode(value)}&"]) : ""
+  # extra_selectors = (length(var.workers[count.index].extra_selectors) > 0) ? "arg=2&" : "arg=1&"
```

Comment:
Why is this commented out? If it's not used, remove it

Reply:
Woops, didn't see I left this in there, just removed it